backend/src/test/java/com/devdive/backend/post/application/adapter/out/PosAdapterTest.java

```diff
@@ -0,0 +1,125 @@
+package com.devdive.backend.post.application.adapter.out;
+
+import com.devdive.backend.post.adapter.out.persistence.*;
+import com.devdive.backend.post.adapter.out.persistence.repository.MemberPostRepository;
+import com.devdive.backend.post.adapter.out.persistence.repository.PostAuthorsRepository;
+import com.devdive.backend.post.adapter.out.persistence.repository.PostRepository;
+import com.devdive.backend.post.application.dto.PostCreateRequestDto;
+import jakarta.persistence.EntityManager;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+
+@DataJpaTest
+@ActiveProfiles("test")
+@Transactional
+public class PosAdapterTest {
+
+    @MockBean
+    MemberPostRepository memberPostRepository;
```

Comment:
모킹하는 것보다는 테스트 픽스처를 직접 넣는게 좋아보입니다.
```suggestion
    @Autowired
    MemberPostRepository memberPostRepository;
```